packages/app/components/EventCard/EventCardRegistration.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { StyleSheet, Button, View } from "react-native";
+import { Icon } from "react-native-elements";
+import FaceIcon from "./FaceIcon";
+
+// Sample array for testing
+const sampleArray = [0,1,2,3];
+
+const styles = StyleSheet.create({
+  buttonView: {
+    flex: 7,
+    justifyContent: 'center',
+    borderRadius: 100
+  },
+  button: {
+    borderRadius: 10,
+    marginBottom: 0,
+    marginLeft: 0,
+    marginRight: 0,
+  },
+  faceView: {
+    flexDirection: 'row',
+    flex: 3,
+    justifyContent: 'center'
+  }
+}); 
+
+export default function EventCardRegistration(props) {
+  const { register } = props;
+  
+  return (
+    <View style={{ flexDirection: 'row' }}>
+      <View style={styles.buttonView}>
+        <Button
+          icon={<Icon color='#3498db' />}
```

Comment:
Is there some problem that forces you to change the button color using an icon instead of with CSS?

Reply:
I only added the color code in there to make it clear for anyone who reads it. It doesn't serve any functional purposes. If you deem that it is unnecessary, I'll remove it.

Reply:
You are already adding CSS styling to the button, so to me, it feels redundant to explicitly add a color code there as its own property instead of being part of the existing button styling.